application/lib/main.dart

```diff
@@ -44,7 +48,17 @@ class _MyAppState extends State<MyApp> {
           homeRoute: (context) => ChangeNotifierProvider(
               create: (context) => HomeViewModel(),
               child: const HomeScreen()),
+          eventRoute: (context) => ChangeNotifierProvider(
+            create: (context) => EventViewModel(),
+            child: Builder(
+              builder: (BuildContext context) {
+                EventModel eventModel = Provider.of<EventModel>(context);
```

Comment:
A providerrel való feloldálst kérlek ne szervezzük ki egészen idáig. Az maradhat egy szinttel lejjebb az EventScreenen, ahogy a HomeScreen esetében is.